### ISSUE TITLE
Atom-typescript is not renaming some symbols correctly when using TypeScript 3

### ISSUE THREAD

**Issue by InExtremaRes** [TypeStrong/atom-typescript#1444]
For instance, in the next snippet, I put the cursor over the `a` parameter, press F2 and rename to `arg`

```ts
const foo = (a: number, b: number) => a + b;
```

After pressing Enter this is the result:

```ts
const foo = (arg: number, b: number) =arg a + b;
```

The expected result (as working in TS 2.9.2) is:

```ts
const foo = (arg: number, b: number) => arg + b;
```

I haven't tested/searching much, so I don't know if the problem is with atom-typescript or in the TS service, but same files/configuration works as expected in TS 2.9.

I am using:
 - atom-typescript: 12.6.2
 - TypeScript: 3.0.1
 - Atom: 1.29.0
 - Electron: 2.0.7

**Comment by Claudisimo** [comment]
I tried to rename with emacs using tide with tsserver 3.0.1 and this doesn't happen. I can reproduce this issue with atom, so the problem should be with atom-typescript i think

**Comment by lierdakil** [comment]
Thanks for reporting and investigating. Should be fixed in v12.6.3.

**Comment by InExtremaRes** [comment]
Perfect! Thank you. It works very well right now.
